refinery/config/settings/base.py

```diff
@@ -658,7 +658,8 @@ def get_setting(name, settings=local_settings, default=None):
 # Time in seconds to wait before killing unused visualization
 DJANGO_DOCKER_ENGINE_SECONDS_INACTIVE = 60 * 60
 # Location of DjangoDockerEngine proxy logging
-PROXY_LOG = '/tmp/django_docker_engine.log'
+DJANGO_DOCKER_ENGINE_DATA_DIR = '/data/django-docker-engine-data'
+PROXY_LOG = '/data/django-docker-engine.log'
```

Comment:
It's probably best to follow [standard Django logging practices](https://docs.djangoproject.com/en/1.7/topics/logging/) instead of restricting logging to just a file.

Reply:
Good point: I think there was some reason initially why it didn't fit, but I can't recall it now, and I'm not sure the reason would be valid even if I could. https://github.com/refinery-platform/django_docker_engine/issues/80